GetBPDCitizen/__tests__/handler.test.ts

```diff
@@ -51,13 +62,21 @@ describe("GetBPDCitizenHandler", () => {
     } as BPDCitizen);
     expect(responseValue.payment_methods).toHaveLength(2);
   });
+describe("GetBPDCitizenHandler", () => {
+  it("should return a success response if the user was found on db", async () => {
+    aSuccessCase(aFiscalCode);
+  });
+
+  it("should return a success response if the user was found on db and a correct support token is provided", async () => {
+    aSuccessCase(aSupportToken);
+  });
```

Comment:
Although I agree there's a lot of code in common, I think we should prefer code duplication when writing tests. This will make every test a kind of ready-to-use usage example. See also [this](https://pagopa.github.io/io-docs/io-handbook/development-styleguide#avoid-wrapping-tests-with-helper-functions).

What we can do to mitigate the duplication problem here is to adopt another mocking strategy: instead of setting the mock implementation on every scenario, make the happy-path implementation the default one, and just set when you want to test wrong path scenarios